packages/strapi-plugin-graphql/services/Query.js

```diff
@@ -175,13 +192,15 @@ module.exports = {
 
       // Plural.
       return async (ctx, next) => {
-        ctx.params = this.amountLimiting(ctx.params);
-        ctx.query = Object.assign(
-          this.convertToParams(_.omit(ctx.params, 'where')),
-          ctx.params.where,
+        const queryOpts = {};
+        queryOpts.params = this.amountLimiting(ctx.params);
+        queryOpts.query = Object.assign(
+          {},
+          this.convertToParams(_.omit(queryOpts.params, 'where')),
+          this.convertToQuery(queryOpts.params.where)
         );
 
-        return controller(ctx, next);
+        return controller(Object.assign({}, ctx, queryOpts, { send: ctx.send }), next, { populate: [] });
```

Comment:
Don't populate any relation in the GraphQL context (Which will make things go faster :yay:)

Reply:
I've already did an update like that in this PR https://github.com/strapi/strapi/pull/2380, we should try to synchronise our work.